woocommerce/api/Abstract_Cacheable_API_Base.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace SkyVerge\WooCommerce\PluginFramework\v5_10_10;
+
+use SkyVerge\WooCommerce\PluginFramework\v5_10_10\API\Cacheable_Request_Trait;
+
+abstract class Abstract_Cacheable_API_Base extends SV_WC_API_Base
+{
+
+
+	/** @var bool whether the response was loaded from cache */
+	protected $response_loaded_from_cache = false;
+
+
+	/**
+	 * Simple wrapper for wp_remote_request() so child classes can override this
+	 * and provide their own transport mechanism if needed, e.g. a custom
+	 * cURL implementation
+	 *
+	 * @since 2.2.0
```

Comment:
Should this be set to when the override was added? Since we're in the context of this class.
```suggestion
	 * @since 5.10.10
```